docs/UserGuide.md

```diff
@@ -95,6 +131,14 @@ For example,
  }]
 ```
 
+For a position,
+
+```json
+[{
+ "title" : "HR Manager"
```

Comment:
I added status in the json as well, so its saved as 
`[{
  "title" : "HR Manager",
  "positionStatus: "OPEN"
}]`

Reply:
Position status JSON field is added